Dockerfile.processing_image

```diff
@@ -21,7 +21,7 @@ RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
  && rm -rf single-cell-explorer client build/client /usr/bin/npm
 
 # Install python dependencies
-RUN pip3 install loompy==3.0.6 scanpy==1.6.0 python-igraph==0.8.3 louvain==0.7.0 tiledb==0.7.4 awscli
```

Comment:
I missed this previously when I was upgrading tiledb versions since I had only searched in `requirements.txt`.

In any case, this tiledb version is replaced below on line 29 from the version specified in `requirements.txt`.